generator-honeycomb/test/app.js

```diff
@@ -197,20 +214,27 @@ describe('generator-honeycomb', () => {
         before((done) => {
           if (type === 'prompts') {
             prompts.templateEngine = 'react';
-            helpers.run(appPath).withPrompts(prompts).on('end', done);
+            helpers
+              .run(appPath)
+              .withPrompts(prompts)
+              .on('end', done);
           } else if (type === 'options') {
             options.template = 'react';
-            helpers.run(appPath).withOptions(options).on('end', done);
+            helpers
+              .run(appPath)
+              .withOptions(options)
+              .on('end', done);
           }
         });
 
         it('creates expected files', () => {
           assert.file([
-            'src/client/components/Example.jsx',
-            'src/client/client.jsx',
-            'src/server/views/index/index.jsx',
+            'src/client/components/Example.js',
+            'src/client/client.js',
+            'src/server/views/index/index.js',
             'test/unit/client/components/Example.test.js',
             'test/unit/client/components/__snapshots__/Example.test.js.snap',
+            'test/.eslintrc.yml',
```

Comment:
This file (`test/.eslintrc.yml`) is global (at the top) and already tested. Please remove this line.